### DISCUSSION TITLE
Zone Editor "Dots" Logic?

### DISCUSSION THREAD

**Discussion by Archer36** [blakeblackshear/frigate#7789] [Ask A Question] [answered]
Hello! Random question that I searched for but couldn't find the right search terms. 

When creating a zone/mask, what is the logic when adding new "dots" to the polygon? Sometimes it seems like when I add a new dot it goes where I double click. Other times it directly opposite of where I double click causing me to have to shuffle the other dots to get one where I need it. 

Thanks!

**Comment by NickM-27** [comment, marked as answer]
You only need to single click for a dot. The dot gets added where it thinks you are trying to create a dot between two others but it is difficult. Generally going in a clockwise direction is the most effective I have found

**Comment by Archer36** [comment]
Got it, that is the helpful tip I was looking for. Thanks!
